applications_tests/lethe-fluid/heat_transfer_heat-flux.prm

```diff
@@ -0,0 +1,172 @@
+# Listing of Parameters
+#----------------------
+
+set dimension = 2
+
+#---------------------------------------------------
+# Simulation Control
+#---------------------------------------------------
+
+subsection simulation control
+  set method           = bdf1
+  set time end         = 0.01
+  set time step        = 0.01
+  set adapt            = false
+  set output name      = heat_flux
+  set output frequency = 1
+end
```

Comment:
Why not instead add an application state with a steady-state solution which is analytical? This would make it easier to validate that the code is fully working well no?

Reply:
It is better, indeed. Just made it.